Library/Formula/llvm.rb

```diff
@@ -116,18 +152,39 @@ def install
     # Apple's libstdc++ is too old to build LLVM
     ENV.libcxx if ENV.compiler == :clang
 
-    if build.with?("lldb") && build.without?("clang")
-      raise "Building LLDB needs Clang support library."
+    # compiler-rt has some iOS simulator features that require i386 symbols
+    # I'm assuming the rest of clang needs support too for 32-bit compilation
+    # to work correctly, but if not, perhaps universal binaries could be
+    # limited to compiler-rt. llvm makes this somewhat easier because compiler-rt
+    # can almost be treated as an entirely different build from llvm.
+    ENV.permit_arch_flags
```

Comment:
I thought that that flag was required because some of the sanitizers that are built for iOS simulator were being built 64-bit only, while the simulator uses both 64 bit and 32 bit, depending on the particular model being simulated. Trying again with a different tweak...